### PR TITLE
Use common_timezone and fix idempotence

### PR DESCRIPTION
**Pull request by spk**
Thanks-to: https://github.com/mtpereira/ansible-common/commit/8e693b3db3b15c440428f378e13ce8d04186d637

cf https://github.com/sovereign/sovereign/pull/422

### CODE REVIEW

File: roles/common/tasks/main.yml [hunk 1]
```diff
@@ -41,15 +41,22 @@
...
+  register: common_timezone_config
 
-- name: Set localtime to UTC
+- name: timezone - Set localtime to UTC
   file: src=/usr/share/zoneinfo/Etc/UTC dest=/etc/localtime
+  when: common_timezone_config.changed
 
-- name: Reconfigure tzdata
-  action: command dpkg-reconfigure -f noninteractive tzdata
-  when: ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu'
```
Comment (spk) [inline roles/common/tasks/main.yml]: Since sovereign is focusing on Debian/Ubuntu I've removed this condition

**Comment by neuhaus** [comment]
Looks good to me but I don't have time to test it

File: vars/defaults.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@
 ###############################################################################
 
 # # common
+# common_timezone: 'Etc/UTC' (required)
```
Comment (ariddell) [inline vars/defaults.yml]: Is there a reason we can't set to UTC as a default?

Comment (spk) [inline vars/defaults.yml]: No don't think so its already currently the default. I will set it as default then.

Comment (ariddell) [inline vars/defaults.yml]: Why mark it (required) then? I think most people would be 

Sorry, I wasn't clear. It's including the line in user.yml that seems
unnecessary -- it implies users need to override it in general. Why not
drop the line from user.yml (i.e., have it be a variable where the
default is fine, as is the case for ``main_user_shell: "/bin/bash"``, which is only
in defaults and not in user.yml)

Comment (spk) [inline vars/defaults.yml]: Yes, see 6330705

**Comment by ariddell** [comment]
This looks good. Thanks!

**Pull request merged**